docs/framework/unmanaged-api/metadata/corelementtype-enumeration.md

```diff
@@ -1,144 +1,147 @@
+    ELEMENT_TYPE_INTERNAL       = 0x21,
+    ELEMENT_TYPE_MAX            = 0x22,
+
+    ELEMENT_TYPE_MODIFIER       = 0x40,
+    ELEMENT_TYPE_SENTINEL       = 0x01 | ELEMENT_TYPE_MODIFIER,
+    ELEMENT_TYPE_PINNED         = 0x05 | ELEMENT_TYPE_MODIFIER
+
+} CorElementType;
+```
+
+## Members
+
+|Member|Description|
+|------------|-----------------|
+|`ELEMENT_TYPE_END`|Used internally.|
+|`ELEMENT_TYPE_VOID`|A void type.|
+|`ELEMENT_TYPE_BOOLEAN`|A Boolean type|
+|`ELEMENT_TYPE_CHAR`|A character type.|
+|`ELEMENT_TYPE_I1`|A signed 1-byte integer.|
+|`ELEMENT_TYPE_U1`|An unsigned 1-byte integer.|
+|`ELEMENT_TYPE_I2`|A signed 2-byte integer.|
+|`ELEMENT_TYPE_U2`|An unsigned 2-byte integer.|
+|`ELEMENT_TYPE_I4`|A signed 4-byte integer.|
+|`ELEMENT_TYPE_U4`|An unsigned 4-byte integer.|
+|`ELEMENT_TYPE_I8`|A signed 8-byte integer.|
+|`ELEMENT_TYPE_U8`|An unsigned 8-byte integer.|
+|`ELEMENT_TYPE_R4`|A 4-byte floating point.|
+|`ELEMENT_TYPE_R8`|An 8-byte floating point.|
+|`ELEMENT_TYPE_STRING`|A System.String type.|
+|`ELEMENT_TYPE_PTR`|A pointer type modifier.|
+|`ELEMENT_TYPE_BYREF`|A reference type modifier.|
+|`ELEMENT_TYPE_VALUETYPE`|A value type modifier.|
+|`ELEMENT_TYPE_CLASS`|A class type modifier.|
+|`ELEMENT_TYPE_VAR`|A class variable type modifier.|
+|`ELEMENT_TYPE_ARRAY`|A multi-dimensional array type modifier.|
+|`ELEMENT_TYPE_GENERICINST`|A type modifier for generic types.|
+|`ELEMENT_TYPE_TYPEDBYREF`|A typed reference.|
+|`ELEMENT_TYPE_I`|Size of a native integer.|
+|`ELEMENT_TYPE_U`|Size of an unsigned native integer.|
+|`ELEMENT_TYPE_FNPTR`|A pointer to a function.|
+|`ELEMENT_TYPE_OBJECT`|A System.Object type.|
+|`ELEMENT_TYPE_SZARRAY`|A single-dimensional, zero lower-bound array type modifier.|
+|`ELEMENT_TYPE_MVAR`|A method variable type modifier.|
+|`ELEMENT_TYPE_CMOD_REQD`|A C language required modifier.|
+|`ELEMENT_TYPE_CMOD_OPT`|A C language optional modifier.|
+|`ELEMENT_TYPE_INTERNAL`|Used internally.|
+|`ELEMENT_TYPE_MAX`|An invalid type.|
+|`ELEMENT_TYPE_MODIFIER`|Used internally.|
+|`ELEMENT_TYPE_SENTINEL`|A type modifier that is a sentinel for a list of a variable number of parameters.|
+|`ELEMENT_TYPE_PINNED`|Used internally.|
+
+## Remarks
+
+The type modifiers form the basis for representing more complex types. A `CorElementType` type modifier value is applied to the value that immediately follows it in the type signature. The value that follows the `CorElementType` type modifier value can be a `CorElementType` simple type value, a metadata token, or other value, as specified in the following table.
+
 > [!NOTE]
->  All numbers (*number*, *argument Count*, *metadata token*, *rank*, *count*, and *bound*) are stored as compressed integers. See [Standard ECMA-335 - Common Language Infrastructure (CLI)](https://go.microsoft.com/fwlink/?LinkID=116487) on the ECMA Web site for details.  
-  
-|Type modifier|Format|  
-|-------------------|------------|  
-|`ELEMENT_TYPE_PTR`|ELEMENT_TYPE_PTR <a `CorElementType` value>|  
```

Comment:
The change from "a" to "an" here is incorrect. Because m is pronounced as "em", with a leading vowel sound, "an" should be used.

Reply:
Not sure if i understood the rule, but it's fixed 😄

Reply:
Thanks, @nschonni. I'll merge your PR now.